linetools/guis/utils.py

```diff
@@ -154,25 +154,21 @@ def set_llist(llist, in_dict=None, sort=True):
                 if llist == 'OVI':
                     gdlines = u.AA*[629.730, 702.332, 770.409, 780.324, 787.711, 832.927, 972.5367, 977.0201,
                         1025.7222, 1031.9261, 1037.6167, 1206.5, 1215.6700, 1260.4221]
-                    llist_cls = LineList('Strong')
-                    llist_cls = llist_cls.subset_lines(gdlines)
+                    llist_cls = LineList('Strong', sort_by=sort_by)
+                    llist_cls = llist_cls.subset_lines(gdlines, sort_by='as_given')
```

Comment:
It can be both. `sort_by='as_given'` is special in the sense that it preserves the order given by, in this case, `gdlines`.